Output tz name, offset, dst, country code, etc.

klinquist opened the pull request.

ironwallaby commented:
Nice patch, thanks.

Gimme some time to ponder whether or not to merge this. I like the idea of adding the convenience abbreviations, but I think everything else is better left to a timezone-aware time library. (In particular, the "dst" and "gmtoffset" fields are completely useless without the full IANA database, since the rules for what the actual offset for a given date is are very, very complex, and supporting them is beyond the scope of this library. I personally recommend using `moment-timezone` for handling that in JS.)

klinquist commented:
Hi,
No problem - perhaps there is too much data there.   I was using the gmtOffset data in a modified version of this to sort for display by the client.   
I removed the extraneous data and added a .list function which outputs the timezone array sorted by GMT offset (based on its "name", which I normalized throughout).
Great for my use case, no worries if you don't want to merge.
I was an original kickstarter backer for DarkSky, btw.  Great app!

ironwallaby commented:
I'd prefer not to include this—I think it's better suited so a separate library. (As noted above, we use `moment-timezone` for this!)
